worldengine/drawing_functions.py

```diff
@@ -40,11 +40,11 @@ def draw_rivers_on_image(world, target, factor=1):
 
     for y in range(world.height):
         for x in range(world.width):
-            if world.is_land((x, y)) and (world.river_map[x, y] > 0.0):
+            if world.is_land((x, y)) and (world.river_map[y, x] > 0.0):
```

Comment:
Was there a reason for that? I changed it in PR #163, as part of unifying the coordinate-usages. It was not just because this inconsistency was bugging me, but when I replaced the image-writer the ancient-maps were a little bit messed up due to the inconsistent coordinates. Instead of working on a workaround I flipped the arrays. (Which was fairly easy, that's why all of this code uses [y, x]-notation. The other way would have been a lot more work.)